style.css

```diff
@@ -21,3 +21,27 @@ video {
 .blogPhoto {
   width: 500px;
 }
+#map {
+  height: 250px;
+  width:600px;
+}
+fieldset.main {
+  margin: 10px 0 10px 0;
+  border: 1px solid black;
+  padding: 10px;
+  width: 600px;
+  background-color: #E8F3EF;
+}
+input[type="text"], input[type="number"], input[type="email"] , input[type="tel"]{
```

Comment:
This is fine, but it's often easier to read if you do this:

```
.selctor-1,
.selector-2,
.etc,
.etc {
  property: value;
}
```

Just a helpful tip!